src/app/features/profile/edit-profile/edit-profile.component.html

```diff
@@ -0,0 +1,472 @@
+            <div class="flex w-full rounded-xl">
+              <input
+                type="text"
+                name="id-check"
+                id="id-check"
+                class="w-full border border-slate-800 bg-slate-50 rounded-xl py-2.5 pr-3 text-gray-900 placeholder:text-gray-400 focus:outline-teal-500 sm:text-sm sm:leading-6"
+                placeholder="0000000"
+                required
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <!-- #3 / about section -->
+
+    <div class="p-4 bg-white mb-2">
+      <label
+        for="about"
+        class="font-bold pb-6 before:absolute before:block before:w-6 before:h-1 before:bg-teal-500 before:rounded-sm before:translate-y-6"
+      >
+        نبذة عني
+      </label>
+      <div class="col-span-full">
+        <div class="mt-2">
+          <textarea
+            id="about"
+            name="about"
+            rows="6"
+            class="w-full rounded-xl leading-6 border border-slate-800 bg-slate-50 pr-4 py-2 placeholder:text-gray-400 sm:text-sm sm:leading-6 focus:outline-teal-500"
+            placeholder="اكتب هنا"
+            style="resize: none"
+          ></textarea>
+        </div>
+      </div>
+    </div>
+
+    <!-- #4 / location section -->
+
+    <div class="section-4 p-4 bg-white mb-2">
+      <h1
+        class="font-bold pb-6 before:absolute before:block before:w-6 before:h-1 before:bg-teal-500 before:rounded-sm before:translate-y-6"
+      >
+        موقعك
+      </h1>
+      <div class="grid grid-cols-1 md:grid-cols-2 gap-4">
+        <div class="bg-white">
+          <p>البلد <span class="text-red-600">*</span></p>
+          <div class="mt-2">
+            <div class="flex w-full rounded-xl">
+              <ng-select
+                formControlName="selectedCountry"
+                class="w-full rounded-xl leading-6 border border-slate-800 pr-4 py-2 placeholder:text-gray-400 sm:text-sm sm:leading-6 focus:outline-teal-500"
+              >
+                <ng-option
+                  *ngFor="let country of countries"
+                  [value]="country.name"
+                >
+                  {{ country.name }}
+                </ng-option>
+              </ng-select>
+            </div>
+          </div>
+        </div>
+        <div class="bg-white">
+          <p>المدينة <span class="text-red-600">*</span></p>
+          <div class="mt-2">
+            <div class="flex w-full rounded-xl">
+              <ng-select
+                formControlName="selectedCity"
+                class="w-full rounded-xl leading-6 border border-slate-800 pr-4 py-2 placeholder:text-gray-400 sm:text-sm sm:leading-6 focus:outline-teal-500"
+              >
+                <ng-option
+                  *ngFor="let city of selectedCountry?.cities"
+                  [value]="city.name"
+                >
+                  {{ city.name }}
+                </ng-option>
+              </ng-select>
```

Comment:
```suggestion
              <ng-select
                formControlName="selectedCity"
                [items]="selectedCountry?.cities ?? []" [bindLabel]="'name'" [bindValue]="'name'"
                class="w-full rounded-xl leading-6 border border-slate-800 pr-4 py-2 placeholder:text-gray-400 sm:text-sm sm:leading-6 focus:outline-teal-500"
              >
              </ng-select>
```

You can use this way in countries, cities and regions